.github/workflows/fast-forward.yml

```diff
@@ -0,0 +1,45 @@
+name: Fast-forward tracking branch for latest CodeQL version
```

Comment:
I would suggest renaming to "selected" CodeQL version since this workflow doesn't try to determine the latest version.  We could add a comment explaining that this is used in the release process to fast forward to the latest version.